hack/build-local-images.py

```diff
@@ -122,16 +122,6 @@
         },
         "files": {}
     },
-    "nginx-router": {
```

Comment:
@rajatchopra do you have any opinion about this?

Reply:
If we have no plan to test nginx then this is fine. We may have to have some way to build and host the nginx image for origin releases though.

Reply:
Once we have nginx image for origin, we could add again to this script if needed.
Currently this change was done to unblock hack/build-local-images.py script, otherwise it tries to build nginx-router, since the image doesn't exists it fails and it won't proceed building other local images.